docs/content/start/telemetry.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Quick Start
+menuPosition: 4
+codeCopyButton: true
+aliases:
+  - "/docs/get-started/telemetry/"
+---
+
+
+With thousands of developers using our framework to build their services and applications, telemetry is an essential part of maintaining and troubleshooting the health of the applications, so we are offering just that! We have integrated a way for the developers to set up a telemetry system by allowing them to create custom logging and handling logics to be passed in and called by our Fluid Framework telemetry pipeline.
+
+## ITelemetryBaseLogger interface
+
+The `ITelemetryBaseLogger` is an interface within the `@fluidframework/common-definitions` package. This interface can be implemented and passed into the `createContainer()` and `getContainer()` methods in `TinyliciousClient` as part of the `TinyliciousContainerConfig` type parameter.
+
+```ts
+export interface TinyliciousContainerConfig {
+  id: string;
+  logger?: ITelemetryBaseLogger;
+}
+```
+
+```ts
+public async createContainer(
+  serviceContainerConfig: TinyliciousContainerConfig,
+  containerSchema: ContainerSchema,
+): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]>
+```
+
+
+### Properties and methods
+
+The interface contains a `supportTags` property and a `send()` method as shown:
+
+```ts
+export interface ITelemetryBaseLogger {
+  /**
+   * An optional boolean which indicates to the user of this interface that tags (i.e. `ITaggedTelemetryPropertyType`
+   * objects) are in use. Eventually this will be a required property, but this is a stopgap that allows older hosts
+   * to continue to pass through telemetry without trouble (this property will simply show up undefined), while our
+   * current logger implementation in `telmetry-utils` handles tags in a separate manner.
+   */
+  supportsTags?: true;
+  send(event: ITelemetryBaseEvent): void;
+}
+```
+
+- SupportsTags
+  - These tags are generic strings used to classify different events. In a simple logger, all events are untagged and handled the same by your logger's implementation. However, in some scenarios, where some data should be handled separately (e.g. private customer data), then it would be worthwhile to "tag" the event with some identifier.
```

Comment:
FYI this is probably going away, and tag support will be assumed.  Documentation about tags would move to a comment documenting what ITelemetryBaseEvent is.